src/main/java/com/dataloom/data/DataApi.java

```diff
@@ -18,47 +18,47 @@
 import retrofit2.http.Path;
 
 public interface DataApi {
-    String CONTROLLER                      = "/data";
+    String CONTROLLER                      = "data";
 
     String FULLQUALIFIEDNAME               = "fqn";
     String NAME                            = "name";
     String NAME_SPACE                      = "namespace";
     String TYPE_NAME                       = "typename";
 
-    String MULTIPLE                        = "/multiple";
-    String ENTITYSET                       = "/entityset";
-    String ENTITY_DATA                     = "/entitydata";
-    String FILTERED                        = "/filtered";
-    String INTEGRATION                     = "/integration";
-    String FULLQUALIFIEDNAME_PATH          = "/{" + FULLQUALIFIEDNAME + "}";
-    String FULLQUALIFIEDNAME_PATH_WITH_DOT = "/{" + FULLQUALIFIEDNAME + ":.+}";
-    String NAME_PATH                       = "/{" + NAME + "}";
-    String NAME_SPACE_PATH                 = "/{" + NAME_SPACE + "}";
-    String TYPE_NAME_PATH                  = "/{" + TYPE_NAME + "}";
-
-    @GET( CONTROLLER + ENTITY_DATA + NAME_SPACE_PATH + TYPE_NAME_PATH + NAME_PATH )
+    String MULTIPLE                        = "multiple";
+    String ENTITYSET                       = "entityset";
+    String ENTITY_DATA                     = "entitydata";
+    String FILTERED                        = "filtered";
+    String INTEGRATION                     = "integration";
+    String FULLQUALIFIEDNAME_PATH          = "{" + FULLQUALIFIEDNAME + "}";
+    String FULLQUALIFIEDNAME_PATH_WITH_DOT = "{" + FULLQUALIFIEDNAME + ":.+}";
+    String NAME_PATH                       = "{" + NAME + "}";
+    String NAME_SPACE_PATH                 = "{" + NAME_SPACE + "}";
+    String TYPE_NAME_PATH                  = "{" + TYPE_NAME + "}";
+
+    @GET( CONTROLLER + "/" + ENTITY_DATA + "/" + NAME_SPACE_PATH + "/" + TYPE_NAME_PATH + "/" + NAME_PATH )
```

Comment:
retrofit doesn't like leading "/" - we have to remove "/" so that it points to the correct relative path.